metadata-ingestion-modules/airflow-plugin/build.gradle

```diff
@@ -19,15 +23,15 @@ task environmentSetup(type: Exec, dependsOn: checkPythonVersion) {
   commandLine 'bash', '-c', "${python_executable} -m venv ${venv_name} && ${venv_name}/bin/python -m pip install --upgrade pip wheel 'setuptools>=63.0.0'"
 }
 
-task installPackage(type: Exec, dependsOn: environmentSetup) {
+task installPackage(type: Exec, dependsOn: [environmentSetup, ':metadata-ingestion:codegen']) {
   inputs.file file('setup.py')
   outputs.dir("${venv_name}")
   // Workaround for https://github.com/yaml/pyyaml/issues/601.
   // See https://github.com/yaml/pyyaml/issues/601#issuecomment-1638509577.
   // and https://github.com/datahub-project/datahub/pull/8435.
   commandLine 'bash', '-x', '-c',
     "${pip_install_command} install 'Cython<3.0' 'PyYAML<6' --no-build-isolation && " +
-    "${pip_install_command} -e ."
+    "${pip_install_command} -e . ${extra_pip_requirements}"
```

Comment:
fyi I made some tweaks to the other build file to fix caching (https://github.com/datahub-project/datahub/pull/8685) - might be worth copying some of those changes over